packages/lib/src/artifacts/ContractFactory.ts

```diff
@@ -0,0 +1,213 @@
+import BN from 'bignumber.js';
+import ZWeb3 from './ZWeb3';
+import decodeLogs from '../helpers/decodeLogs';
+import { getSolidityLibNames, hasUnlinkedVariables } from '../utils/Bytecode';
+
+interface ContractSchema {
+  contractName:string;
```

Comment:
Just an OCD here, but I really prefer if we add a space before types from the beginning of this beautiful and grateful process of typscripting the repo:
```suggestion
  contractName: string;
```

Reply:
Sure. If you want to change the linter setting now, could you please modify the existing .ts files to use the convention, so that we don't get an explosion of linting errors all around.
Otherwise, you can add a TS-TODO note on the linter configuration (root/tsling.json) to change it later when we revisit the compiler and linter settings in Phase 5 of the plan.